docs/mp/reactivemessaging/introduction.adoc

```diff
@@ -525,11 +436,104 @@ every messaging channel to have its own probe.
 }
 ----
 
-Caution: Due to the nack support are exceptions thrown in messaging methods NOT translated to error and cancel signals implicitly anymore
+CAUTION: Due to the nack support are exceptions thrown in messaging methods NOT translated to error and cancel signals implicitly anymore
```

Comment:
I'm not sure I understand this? 
Due to the nack support, exceptions are thrown in messaging methods that are no longer translated implicitly to error and cancel signals.

Reply:
@danielkec is your caution statement good as-is or do you want me to rewrite it? This is ready for approval after this issue.